service_provider.go

```diff
@@ -35,13 +36,18 @@ func (receiver *ServiceProvider) Boot(app foundation.Application) {
 
 	if ConfigFacade = app.MakeConfig(); ConfigFacade == nil {
 		color.Errorln(errors.ConfigFacadeNotSet.SetModule(module))
+		shutdownOnCriticalError("ConfigFacade is not set")
```

Comment:
I think it's unnecessary for now, we want to print a warning instead of interrupting.

Reply:
Need to remove the function totally.